optimizer/optimizations.md

```diff
@@ -23,8 +22,40 @@ module EVM-OPTIMIZATIONS
     imports INT-SIMPLIFICATION
 
 
-// {OPTIMIZATIONS}
-
+    claim
+    [optimized.add]:
+      <kevm>
+        <k>
+          ( #next[ ADD ] => .K ) ...
+        </k>
+        <schedule>
+          SCHED
+        </schedule>
+        <useGas>
+          USEGAS
+        </useGas>
+        <ethereum>
+          <evm>
+            <callState>
+              <wordStack>
+                ( W0 : W1 : WS => chop( ( W0 +Int W1 ) ) : WS )
```

Comment:
Why is this added here?

We already have a test-harness that takes the rules in this file and discharges them as claims directly.

Reply:
Oh, actually, this entire directory isn't used at all anymore. We really should delete it, but I left it for illustrative purposes. Any reason to add this?

Reply:
Removed.